webpack.js

```diff
@@ -3,13 +3,46 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 const path = require('path')
+const WebpackSPDXPlugin = require('./build-js/WebpackSPDXPlugin.js')
 const webpackConfig = require('@nextcloud/webpack-vue-config')
 
+const isDev = process.env.NODE_ENV === 'development'
+
 webpackConfig.entry = {
 	admin: path.join(__dirname, 'src', 'admin.js'),
 	user: path.join(__dirname, 'src', 'user.js'),
 	public: path.join(__dirname, 'src', 'public.js'),
 	registration: path.join(__dirname, 'src', 'registration.js'),
 }
 
+// Generate reuse license files if not in development mode
+if (!isDev) {
+	config.plugins.push(new WebpackSPDXPlugin({
```

Comment:
```suggestion
	webpackConfig.plugins.push(new WebpackSPDXPlugin({
```

Reply:
added to https://github.com/nextcloud/terms_of_service/pull/977/commits/4e22755174ff0be03c0f4929b67f2d92eb0b5d0c